lib/fakefs/dir.rb

```diff
@@ -104,13 +104,13 @@ def self.glob(pattern, _flags = 0, &block)
         end.sort
       end
 
-      if pattern.is_a? Array
-        files = pattern.map do |matcher|
-          matches_for_pattern.call matcher
-        end.flatten
-      else
-        files = matches_for_pattern.call pattern
-      end
+      files = if pattern.is_a?(Array)
```

Comment:
prefer this to stay with 2-space indent and don't waste whitespace or having to adjust whitespace when var name changes
```
files = 
  if dfsdfsfd
```